syntax = "proto3";

package inventory;

service InventoryService {
  rpc UpdateInventory(InventoryUpdateRequest) returns (InventoryUpdateResponse);
}

message InventoryUpdateRequest {
  string product_id = 1;
  int32 quantity = 2;
  string action = 3; // "add" or "remove"
}

message InventoryUpdateResponse {
  string message = 1;
}
